background.js

```diff
@@ -0,0 +1,57 @@
+function getHash() {
+    
+    var url = document.location.href;
+    const examplePRURL = "https://github.com/sharpspring/frontend/pull/#####";
+
+    //Get the PR number from the URL assuming we're on the conversation tab
+    function getPR(url){
+        const regex = new RegExp('/pull/');
+        const index = url.search(regex);
+        var prNum = url.slice([index+6],url.length);
+        prNum = parseInt(prNum)
+        return(prNum)
+    }
+    
+    //If we're on the Commits tab
+    if (url.startsWith("https://github.com/sharpspring/") && url.endsWith("/commits")){
```

Comment:
I think we just need to remove` && url.endsWith("/commits")` and the conversations tab code